crates/swc_ecma_transforms_optimization/src/simplify/expr/tests.rs

```diff
@@ -15,6 +15,7 @@ fn fold(src: &str, expected: &str) {
 
             chain!(
                 resolver(unresolved_mark, top_level_mark, false),
+                paren_remover(None),
```

Comment:
We have added the `paren_remover` here, and I suppose we need a `fixer` for that. @kdy1

Reply:
Never mind. This is a test pass